etl/steps/data/garden/artificial_intelligence/2023-06-14/ai_all_conferences.py

```diff
@@ -0,0 +1,55 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from typing import cast
+
+import pandas as pd
+from owid.catalog import Dataset, Table
+from structlog import get_logger
+
+from etl.helpers import PathFinder, create_dataset
+
+log = get_logger()
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    log.info("ai_conference_type.start")
+
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow_ai_small = cast(Dataset, paths.load_dependency("ai_small_conferences"))
+    ds_meadow_ai_large = cast(Dataset, paths.load_dependency("ai_large_conferences"))
+    ds_meadow_ai_total = cast(Dataset, paths.load_dependency("ai_conferences_total"))
+
+    # Read table from meadow dataset.
+    tb_small = ds_meadow_ai_small["ai_small_conferences"]
+    tb_large = ds_meadow_ai_large["ai_large_conferences"]
+    tb_total = ds_meadow_ai_total["ai_conferences_total"]
+
+    df_small = pd.DataFrame(tb_small)
+    df_large = pd.DataFrame(tb_large)
+    df_total = pd.DataFrame(tb_total)
+
+    df_total["conference"] = "Total"
+    df_small_tot = pd.concat([df_small, df_total], axis=0, join="inner").reset_index(drop=True)
+    df_all = pd.concat([df_small_tot, df_large], axis=0, join="inner").reset_index(drop=True)
+
+    df_all["number_of_attendees__in_thousands"] = df_all["number_of_attendees__in_thousands"].apply(
+        lambda x: round(x * 1000)
+    )
```

Comment:
I think you can also

```python
df_all["number_of_attendees__in_thousands"] = (1000 * df_all["number_of_attendees__in_thousands"]).round()
```

Haven't tested, but might be more efficient.

Reply:
thanks! yep, not sure why I did this haha :)